examples/react-vite/src/providers.tsx

```diff
@@ -122,7 +191,7 @@ const WalletInfoComponent = () => {
                 <p>
                   Transaction Hash:{" "}
                   <a
-                    href={`https://sepolia.etherscan.io/tx/${transactionHash}`}
+                    href={`https://sepolia.${CHAIN_CONFIGS[chainId].explorer}/tx/${transactionHash}`}
```

Comment:
I'd prefer if `${CHAIN_CONFIGS[chainId].explorer}` was the entire start of the domain. This is more generic and accounts for cases where the block explorer might not have `sepolia` as a subdomain or the testnet may not be called sepolia.
```suggestion
                    href={`${CHAIN_CONFIGS[chainId].explorer}/tx/${transactionHash}`}
```

Reply:
done